v3.1/reference/transactions/transaction-optimistic.md

```diff
@@ -0,0 +1,178 @@
+---
+title: TiDB 乐观事务模型
+summary: 了解 TiDB 的乐观事务模型。
+category: reference
+aliases: ['/docs-cn/v3.1/reference/transactions/transaction-model/']
+---
+
+# TiDB 乐观事务模型
+
+TiDB 默认使用乐观事务模型，即事务提交时假定不会发生并发冲突，只有在事务最终提交时才会检测冲突，而不是像 MySQL 一样使用行锁来避免写写冲突。
+
+本文介绍 TiDB 乐观事务的原理，以及相关特性。本文假定你对 [TiDB 的整体架构](/v3.1/architecture.md#tidb-整体架构)、[Percolator](https://www.usenix.org/legacy/event/osdi10/tech/full_papers/Peng.pdf) 事务模型以及事务的 [ACID 特性](/v3.1/glossary.md#acid)都有一定了解。
```

Comment:
Suggest moving this para to the beginning of the body part. Keeping the current 1st para closer to the note would be more friendly.

Reply:
@lilin90 Updated in [d2f7b5d](https://github.com/pingcap/docs-cn/pull/2198/commits/d2f7b5deb4c7783eddd666ee713029266b9b8231).